packages/orbit-components/transforms/__testfixtures__/translations-new.input.js

```diff
@@ -13,6 +13,8 @@ import Breadcrumbs, { BreadcrumbsItem } from "@kiwicom/orbit-components/lib/Brea
 /* prettier-ignore */
 
 const Component = () => {
+  const intl = useIntl();
```

Comment:
Should this be on the input? 🤔 shouldn't the codemod handle this?

Reply:
I struggled to properly find a parent nodes, now should be fine 🤞🏻